src/libraries/System.ComponentModel.Composition/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+[assembly: SkipOnMono("Flaky tests: https://github.com/mono/mono/issues/16417")]
```

Comment:
should be

```suggestion
[assembly: ActiveIssue("Flaky tests: https://github.com/mono/mono/issues/16417", TestRuntimes.Mono)]
```

Reply:
`Attribute 'ActiveIssue' is not valid on this declaration type. It is only valid on 'class, method' declarations.`

Reply:
Hm interesting. I don't see a good reason not to allow `ActiveIssue` on assembly, will file a PR in dotnet/arcade.